prisma/schema.prisma

```diff
@@ -13,38 +13,47 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-//May need to update model for photos
-model Hotel {
-  id                   String    @id @default(auto()) @map("_id") @db.ObjectId
-  name                 String
-  location             Location
-  amenities            String[]
-  pricePerNight        Int
-  ratings              Ratings?
-  roomsAvailable       Int
-  description          String
-  singleRoomsAvailable Int
-  doubleRoomsAvailable Int
-  createdAt            DateTime?
+model User {
```

Comment:
It's easy to get user info from auth() on the client side. We should only store data that's needed. In this case, we want an id, an email to query by, and points to track.

I would not store anything more without a good reason.